bitflow/sinksteps.py

```diff
@@ -375,7 +373,7 @@ def print_metrics(self,sample):
 			for k,v in sample.tags.items():
 					t_str += ", {}={}".format(k,v)
 				
-		print("{} {} {}".format(sample.timestamp,t_str,s_str))
+		print("{} {} {}".format(sample.get_printable_timestamp(),t_str,s_str))
```

Comment:
Is this a printf debug output, or a log message? Pls use logging for log messages, and debug outputs should not be in the pull request code

Reply:
this is a pure write on the console. like -